packages/amplication-client/src/Components/CompleteSignupDialog.tsx

```diff
@@ -11,12 +11,10 @@ import "./CompleteSignupDialog.scss";
 const CLASS_NAME = "complete-signup-dialog";
 
 type Props = {
-  handleDialogClose: () => void;
+  onConfirm: () => void;
 };
 
-export const CompleteSignupDialog: React.FC<Props> = ({
-  handleDialogClose,
-}) => {
+export const CompleteSignupDialog: React.FC<Props> = ({ onConfirm }) => {
```

Comment:
Move the file to ./User/ with the rest of the components related to the preview signup flow